src/structures/ThreadChannel.js

```diff
@@ -234,6 +234,22 @@ class ThreadChannel extends Channel {
     return this.parent?.permissionsFor(memberOrRole) ?? null;
   }
 
+  /**
+   * Fetches the owner of this thread
+   * @param {FetchOwnerOptions} [options] The options for fetching the member
```

Comment:
I'm not sure about this. If a user clicks on this, they'll see the description:
> Options used to fetch the owner of guild

This description isn't accurate. I believe the wording should change to something like:
> Options used to fetch the owner of guild or the owner of a thread

Reply:
I don't even know why this typedef exists since it's just BaseFetchOptions but sure, I'll add that